### PR TITLE
Cosmic renaming

### PR DESCRIPTION
**Pull request by soleti**
I propose the following changes to the Cosmic FCL file names to agree with the MDC2020 conventions. @oksuzian might want to comment.

### CODE REVIEW

File: JobConfig/cosmic/S2ResamplerHigh.fcl [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
 physics.filters.filterS1.cutEDepMin: 16
 physics.filters.filterS1: {
          module_type: FilterCosmicsStage1
          inputs : [ "dsResample:CRV" ]
-         cutEDepMin: 16  // Filter events that deposit lower than 16 MeV in CRV 
+         cutEDepMin: 16  // Filter events that deposit lower than 16 MeV in CRV
 }
 physics.TriggerPath: [dsResample, filterS1, @sequence::Primary.TriggerSequence]
 physics.filters.DetStepFilter.MaximumPartMom: 500
-outputs.Output.fileName        : "sim.owner.cosmic-s2-resampler-hi.version.sequencer.art"
```
Comment (oksuzian) [inline JobConfig/cosmic/S2ResamplerHigh.fcl:13]: Can we name it as:
sim.owner.CosmicHi.version.sequencer.art

File: JobConfig/cosmic/S2ResamplerLow.fcl [hunk 2]
```diff
@@ -0,0 +1,6 @@
+// Re-sample cosmic dsstops particles from stage-1
+// Yuri Oksuzian, 2020
+
+#include "JobConfig/cosmic/S2Resampler.fcl"
+outputs.Output.fileName        : "sim.owner.Cosmic.version.sequencer.art"
```
Comment (oksuzian) [inline JobConfig/cosmic/S2ResamplerLow.fcl:5]: Can we name it as:
sim.owner.CosmicLo.version.sequencer.art

**Review by oksuzian — CHANGES_REQUESTED** [review]
Roberto, thank you for doing this!
Please consider making minor changes to file names.

**Review by brownd1978 — APPROVED** [review]

**Review by oksuzian — APPROVED** [review]
Looks good. Thanks, Roberto.

**Pull request merged**